tests/test_ninja.py

```diff
@@ -16,3 +17,7 @@ def _run(program, args):
 
 def test_ninja_module():
     _run("ninja", ["--version"])
+
+
+def test_ninja_package():
+    subprocess.check_call(["python", "-m", "ninja", "--version"])
```

Comment:
Technically better to use `sys.executable` rather than `python` - it will make the tests immune to different $PATH settings.

Reply:
Thanks for the suggestion